### PR TITLE
docs: Native Stack - header options

### PR DESCRIPTION
**Pull request by stanleyoos**
React Navigation, Native Stack, v7.x

added screenshots/videos for: 

- headerTintColor
- headerBackground
- headerBlurEffect
- headerLargeStyle
- headerLargeTitle
- headerLargeTitleStyle
- headerShadowVisible
- headerStyle
- headerLeft
- headerRight

In headerBlurEffect, the value _extraLight_ seems to have no effect, [issue](https://github.com/react-navigation/react-navigation/issues/12096) for that

### CODE REVIEW

**Comment by satya164** [comment]
image

The `extraLight` blur effect doesn't seem to have a blur

File: static/assets/7.x/native-stack/headerBlurEffect-extraLight.png
[the remark this answers is not in the document]
Comment (stanleyoos) [reply static/assets/7.x/native-stack/headerBlurEffect-extraLight.png:1]: It's a bug, every value works except for `extraLight`

**Review by kacperkapusciak — APPROVED** [review]

File: static/assets/7.x/native-stack/headerBlurEffect-extraLight.png
[the remark this answers is not in the document]
Comment (satya164) [reply static/assets/7.x/native-stack/headerBlurEffect-extraLight.png:1]: Make sure to report any bugs you encounter - you can post on discord.

[the remark this answers is not in the document]
Comment (stanleyoos) [reply static/assets/7.x/native-stack/headerBlurEffect-extraLight.png:1]: Sure, I created new issue: [link](https://github.com/react-navigation/react-navigation/issues/12096)

**Pull request merged**